docs/whats-new.asciidoc

```diff
@@ -13,113 +13,126 @@ Other versions: {security-guide-all}/8.13/whats-new.html[8.13] | {security-guide
+=== Use AI Assistant when writing rule queries
 
-[role="screenshot"]
-image::whats-new/images/8.14/contribution-scores-per-alert.png[Contribution scores for top 10 alerts, 90%]
+When creating rules, use AI Assistant to improve rule queries or to quickly correct them.
 
 [float]
-== Detection rules and alerts enhancements
+=== Bulk update custom highlighted fields for rules
 
+Bulk add or remove {security-guide}/rules-ui-create.html#rule-ui-advanced-params[custom highlighted fields] for multiple detection rules.
 
 [float]
-=== Value list improvements
-
-You can now {security-guide}/value-lists-exceptions.html#edit-value-lists[edit value lists] from the UI, wherever you use them. For example, you can now add items to a value list while creating a rule exception that references that value list.
+=== Preview entities and alerts in the alert details flyout
 
-[role="screenshot"]
-image::whats-new/images/8.14/edit-value-lists.png[Edit items in a value list, 90%]
+You can now preview host and user details from the **Insights** tab of the {security-guide}/view-alert-details.html[alert details flyout] instead of going to the **Hosts** or **Users** pages for more information. From the **Correlations** tab in the flyout, you can also preview alerts that are related to each other instead of leaving the flyout to access them.
 
 [float]
-=== Add ES|QL fields as custom highlighted fields
+=== The expandable alert details flyout is enabled by default
 
-When adding custom highlighted fields to an {esql} rule, you can now {security-guide}/rules-ui-create.html#custom-highlighted-esql-fields[specify any fields returned by the rule's query]. This allows you to surface fields that contain useful information for investigating alerts.
+The expandable alert details flyout is now enabled by default in multiple places throughout the {security-app}.
 
 [float]
-=== Editable setup guide field for detection rules
+== Timeline enhancements
 
-You can now {security-guide}/rules-ui-create.html#rule-ui-advanced-params[edit the **Setup guide** field] for user-created custom rules. Use this informational field to list rule prerequisites such as required integrations, configuration steps, and anything else needed for the rule to work correctly.
+[float]
+===  Toggle row renderers on and off in Timeline
+
+Within Timeline, quickly add or remove context from events by toggling row renderers.
 
 [role="screenshot"]
-image::whats-new/images/8.14/setup-guide-field.png[Setup guide field]
+image::whats-new/images/8.15/timeline-ui-renderer.png[Example Timeline with the event renderer highlighted]
 
 [float]
-=== Alert suppression improvements 
+=== Notes moved to a new flyout in Timeline
+
+TBD
```

Comment:
Whoops, I must have started reviewing before this is ready. I'll go ahead and submit to add my other comment.

Reply:
There are a couple of outstanding items but reviews are still welcome :)